aci/resource_aci_vmmctrlrp.go

```diff
@@ -0,0 +1,615 @@
+package aci
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceAciVMMController() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAciVMMControllerCreate,
+		Update: resourceAciVMMControllerUpdate,
+		Read:   resourceAciVMMControllerRead,
+		Delete: resourceAciVMMControllerDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciVMMControllerImport,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"vmm_domain_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"annotation": &schema.Schema{
+				Type: schema.TypeString,
+
+				Optional:    true,
+				Computed:    true,
+				Description: "Mo doc not defined in techpub!!!",
+			},
```

Comment:
Annotation attribute is not required here, because I added it into the base attribute schema